lib/ProductOpener/Ingredients.pm

```diff
@@ -1399,7 +1399,7 @@ INFO
 		$product_ref->{$tagtype . '_tags'} = [];		
 				
 		# skip palm oil classes if there is a palm oil free label
-		if (($class =~ /palm/) and ($product_ref->{labels_tags} ~~ 'en:palm-oil-free')) {
+		if (($class =~ /palm/) and ($product_ref->{labels_tags} == 'en:palm-oil-free')) {
```

Comment:
I think this probably also needs to be an array search, because `labels_tags` is an array? @stephanegigandet

Reply:
yes, labels_tags is an array

We have a has_tag function:  https://github.com/openfoodfacts/openfoodfacts-server/blob/master/lib/ProductOpener/Tags.pm#L179 

has_tag($product_ref, "labels", 'en:palm-oil-free')